core/src/main/antlr/SubstraitType.g4

```diff
@@ -99,83 +99,83 @@ SingleQuote: '\'';
 
 
 Number
-  :  '-'? Int
+  : '-'? Int
   ;
 
 Identifier
   : ('a'..'z' | 'A'..'Z' | '_' | '$') ('a'..'z' | 'A'..'Z' | '_' | '$' | Digit)*
   ;
 
 LineComment
-	:	'//' ~[\r\n]* -> channel(HIDDEN)
-	;
+  : '//' ~[\r\n]* -> channel(HIDDEN)
+  ;
```

Comment:
I'm choosing spaces over tabs here because that it what is used in examples in the ANTLR repo
https://github.com/antlr/grammars-v4/tree/master